foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/ConfigStoreInit.java

```diff
@@ -39,20 +42,41 @@ public class ConfigStoreInit {
     @PostConstruct
     public void init() {
         logger.info("Initialize store configuration properties and policies...");
-        // We are loading all the properties upfront. In the future it might be more advantageous to load them as needed
-        String basePropertyUri = getBootstrapConfiguration("KRAUSENING_BASE");
-        String environmentPropertyUri = getBootstrapConfiguration("KRAUSENING_EXTENSIONS");
-        String basePolicyUri = getBootstrapConfiguration("BASE_POLICY_URI");
-        String environmentPolicyUri = getBootstrapConfiguration("ENVIRONMENT_POLICY_URI");
+
+        // Get users desired storage class. If not set then default to Krausening
+        String storageClass = getBootstrapConfiguration("STORAGE_CLASS") != null ?
+                getBootstrapConfiguration("STORAGE_CLASS") : DaoClass.krausening.name();
+        String propertyDaoClass = EnumUtils.isValidEnum(DaoClass.class, storageClass)?
+                DaoClass.valueOf(storageClass).getValue() : storageClass;
+
+        logger.info("Using property Dao class: {}", propertyDaoClass);
 
         try {
             ConfigLoader configLoader = CDI.current().select(ConfigLoader.class,new Any.Literal()).get();
+            configLoader.setPropertyDaoClass(propertyDaoClass);
             if (configLoader.isFullyLoaded()) {
-                logger.info("Properties are already fully loaded. Skipping reload.");
+                logger.info("Properties and polices are already fully loaded. Skipping reload.");
                 status = Status.LOAD_SKIPPED;
                 return;
             }
 
+            loadConfigs(configLoader);
+            loadPolicies(configLoader);
+
+            logger.info("Successfully initialized store configuration properties and policies.");
+            status = Status.LOAD_COMPLETE;
+            configLoader.updateLoadStatus(true);
```

Comment:
i: load status is used to skip loading properties and polices. Previously it was only set to skip after the properties were loaded. Updated to set to skip after both properties and policies are loaded